### PR TITLE
Fix failing build (install missing READR package) and disable progress when installing pandoc

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

**Comment by msevestre** [comment]
Where does this PAT comes from? 
I have fixed the build by downloading specific version of our components for now
Vignettes build is failing /cc @pchelle  This is why the build is red now

File: appveyor.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases/download/v10.0.72/ospsuite_10.0.72.zip', repos = NULL, type = 'binary')"
   - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/lq7s2r98966wqpjh/artifacts/tlf_1.2.33.zip', repos = NULL, type = 'binary')"
```
Comment (msevestre) [inline appveyor.yml:38]: @Yuri05 I am installing the packages as zip here. THat;s why it works?

**Comment by Yuri05** [comment]
>Where does this PAT comes from?
I have fixed the build by downloading specific version of our components for now
Vignettes build is failing /cc @pchelle This is why the build is red now

The PAT is my GH token encrypted via AppVeyour. I think any valid GH token would do here.
Vignettes were failing due to the missing package readr. Not sure why it was not installed with other dependencies (missing entry in DESCRIPTION?)

With that, the build process runs through.

File: appveyor.yml (hunk 1 above)
Comment (Yuri05) [reply appveyor.yml:38]: Not sure I understand the question...

**Comment by msevestre** [comment]
yes but the PAT is not required here because I am installing the package directly
I had tried this (my own PAT) but it did not work either. So if you want to be sure that it works, you need to revert my changes

File: appveyor.yml (hunk 1 above)
Comment (msevestre) [reply appveyor.yml:38]: THE PAT File is only used when installing from github which I have removed a couple of hours ago
So the PAT is defacto not used here (as you can see, tlf and ospsuite-r are installed from packages and not from github. So the PAT changes are useless
To test if this works, you need to undo my changes from previous commit in develop
Also personal PAT expires after 30 days? This is not sustainable

**Review by msevestre — CHANGES_REQUESTED** [review]

**Comment by Yuri05** [comment]
>THE PAT File is only used when installing from github which I have removed a couple of hours ago
So the PAT is defacto not used here (as you can see, tlf and ospsuite-r are installed from packages and not from github. So the PAT changes are useless

OK, I see. I missed that commit (no PR?)
Will remove the PAT then.

**Comment by Yuri05** [comment]
@msevestre ok, PAT removed and still works - very nice :)

**Comment by pchelle** [comment]
> 
> 
> @msevestre @pchelle
> 
> > Where does this PAT comes from?
> > I have fixed the build by downloading specific version of our components for now
> > Vignettes build is failing /cc @pchelle This is why the build is red now
> 
> The PAT is my GH token encrypted via AppVeyour. I think any valid GH token would do here.
> 
> Vignettes were failing due to the missing package readr. Not sure why it was not installed with other dependencies (missing entry in DESCRIPTION?). So I installed it manually. With that, the build process runs through.

Did we know which function was using `readr` package ? (If I recall, only `ospsuite` was using `readr`)

**Review by pchelle — APPROVED** [review]
Just to make sure, does using the .zip also means we'll have to update the version of `tlf` and `ospsuite` every time they are updated ?

**Comment by Yuri05** [comment]
OK, merging for now.

>Just to make sure, does using the .zip also means we'll have to update the version of tlf and ospsuite every time they are updated ?

In principle yes.
@msevestre we could also create timely unrestricted PAT and try with it. 
Or what we also could do: upload the .zip artifact as a copy without version number. E.g. not only **tlf_1.2.3.zip** but additionally **tlf.zip** (which is just a renamed version of **tlf_1.2.3.zip**). And then use the [permalink to the latest successfully build artifact](https://www.appveyor.com/docs/packaging-artifacts/#permalink-to-the-last-successful-build-artifact) in the appveyor.yml

**Pull request merged**